src/operators/mod.rs

```diff
@@ -10,17 +10,16 @@
 //!   [`crate::window::WindowLogic`].
 
 use crate::pyo3_extensions::{TdPyAny, TdPyCallable, TdPyIterator};
-use crate::recovery::StateBytes;
-use crate::recovery::StateUpdate;
-use crate::recovery::StatefulLogic;
-use crate::window::WindowLogic;
 use crate::try_unwrap;
 use crate::{log_func, unwrap_any};
 use log::debug;
 use pyo3::exceptions::PyTypeError;
 use pyo3::prelude::*;
-use std::task::Poll;
-use std::time::Duration;
+
+pub(crate) mod fold_window;
```

Comment:
🤔 would it make sense for us to further compartmentalize stateful vs not stateful operators? Or too much?